src/OSPSuite.Presentation/UICommands/RunMultipleParameterIdentificationsUICommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OSPSuite.Core.Domain.ParameterIdentifications;
+using OSPSuite.Core.Domain.Services.ParameterIdentifications;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Core.Services;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OSPSuite.Presentation.UICommands
+{
+    public class RunMultipleParameterIdentificationsUICommand : ActiveObjectUICommand<IReadOnlyList<ParameterIdentification>>
```

Comment:
alignment is off. Please run resharper on file

Reply:
done